src/index.css

```diff
@@ -90,3 +106,18 @@ code {
 :root.theme-light code {
 	--color-bg: var(--color-gray-light);
 }
+
+.delete-btn {
+	box-sizing: border-box;
+	background-color: unset;
+	border: none;
+}
+
+.delete-icon {
+	height: 1.5em;
+}
```

Comment:
I think this size looks great. I'd just doublecheck the sizing once the NavBar has been merged to make sure they have a similar weight.